releasing/docker-run.sh

```diff
@@ -60,6 +60,13 @@ case "$TARGET_OS" in
     # Compiler
     [ "$TARGET_VERSION" = xenial ] || [ "$TARGET_VERSION" = bionic ] && _version="$TARGET_VERSION" || _version="common"
     cat ${TARGET_OS}/${_version}/compiler.dockerstage >> ${OUT_DIR}/Dockerfile
+    
+    # Use local flex/bison on Jammy
+    if [ "$TARGET_VERSION" = jammy ]; then
+      BAZEL_OPTS="${BAZEL_OPTS} --//bazel:use_local_flex_bison"
+      cat ${TARGET_OS}/${TARGET_VERSION}/flex_bison.dockerstage >> ${OUT_DIR}/Dockerfile
```

Comment:
I would suggest hardcoding `jammy` in this line, instead of using the TARGET_VERSION envvar.

Reply:
Moreover, since it's a one-liner, you might avoid the `.dockerstage` file:

```
echo 'RUN apt-get install -y flex bison' >> ${OUT_DIR}/Dockerfile
```